.gitignore

```diff
@@ -4,6 +4,7 @@ node_modules
 # build outputs
 dist
 build
+!.github/**/
```

Comment:
Do we need this?

Reply:
We don't anymore, thank you for catching that, and thank you for the review!